test/train-sets/ref/cbe_cover_e_dec.stderr

```diff
@@ -1,22 +1,23 @@
 final_regressor = models/cover_e_dec.model
+warning: currently, mtr is only used for the first policy in cover, other policies use dr
 Num weight bits = 18
 learning rate = 0.5
 initial_t = 0
 power_t = 0.5
 using no cache
 Reading datafile = train-sets/cb_explore_cover.dat
 num sources = 1
-Enabled reductions: gd, scorer, csoaa, cb, cb_explore
+Enabled reductions: gd, scorer, csoaa_ldf, cb_adf, cb_explore_adf_cover, shared_feature_merger, cb_to_cbadf
 average  since         example        example  current  current  current
 loss     last          counter         weight    label  predict features
 0.000000 0.000000            1            1.0  1:0:0.5 1:0.500000        3
-0.323483 0.646967            2            2.0  2:1:0.5 1:0.500000        3
-0.456686 0.589888            4            4.0  1:2:0.5 2:0.591752        3
-0.542755 0.628824            8            8.0  2:1:0.5 2:0.732739        3
+0.500000 1.000000            2            2.0  2:1:0.5 1:0.500000        3
+0.841752 1.183503            4            4.0  1:2:0.5 1:0.591752        3
+0.670876 0.500000            8            8.0  2:1:0.5 1:0.666667        3
```

Comment:
why do these change?

Reply:
Ok, this changes for two reasons.

1) The left side of the diff should be compared with the new file right below this one called `test/train-sets/ref/cbe_cover_e_dec_legacy.stderr`. If we compare those, there is a difference due to a change introduced in this PR where we change the loss function to use IPS estimate instead of doubly robust. See the change in `cb_explore.cc` under `generic_output_example` fn.

2) If you are asking why the left side and the right side are different, it uses a different stack (cb vs cb_adf) + the loss function change mentioned on the previous point.

@pmineiro feel free to chime in in case I'm misremembering anything